sensu_handler_pagerduty_alternative.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+"""
+This module provides a handler for use with Sensu Go (https://sensu.io/)
+
+The intent of this module was to provide a mechanism for being able to pass a data structure
+(not just a string) through to the details of a PagerDuty alert.
+
+While similar to the PagerDuty handler from the folks as Sensu, there are some differences in the
+interface and attention should be paid to the documentation when using it.
+"""
+
+# pylint disable=c0103
```

Comment:
What variable name is this getting disabled for?

Reply:
Actually no longer needed.  :D  I'll remove that.